### PR TITLE
Docs for text and heading

### PR DESCRIPTION
**Pull request by hadasfa**

### CODE REVIEW

File: src/components/Heading/__stories__/Heading.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+</Canvas>
+
+## Import
+
+The Heading component has been introduced in version 2.0 of Vibe.
+However, the import statement `{Heading}` from 'monday-ui-react-core' is already reserved for the <Link href="/?path=/docs/text-legacyheading--overview">LegacyHeading</Link> component.
+Therefore, the new Heading with the updated API can be accessed using the following import statement:
+
+```javascript
+import { Heading } from "monday-ui-react\next";
```
Comment (SergeyRoyt) [inline src/components/Heading/__stories__/Heading.stories.mdx:67]: `monday-ui-react/next`

Comment (SergeyRoyt) [inline src/components/Heading/__stories__/Heading.stories.mdx:67]: `monday-ui-react-core/next`

File: src/components/Heading/__stories__/Heading.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+```
+
+In the upcoming major release of Vie packages (3.0), we will remove LegacyHeading from the main entry, and then you will be able to import this Heading component
```
Comment (SergeyRoyt) [inline src/components/Heading/__stories__/Heading.stories.mdx:70]: `Vibe package`

File: src/components/Heading/__stories__/Heading.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+  </Story>
+</Canvas>
+
+### Overflow
+
+Our Heading component supports overflow state.
+When the text is longer than its container and the ellipsis flag is on, the end of the text will be truncated and will display "..."
+
+<Canvas>
+  <Story name="Overflow" play={titleOverflowSuite}>
```
Comment (SergeyRoyt) [inline src/components/Heading/__stories__/Heading.stories.mdx:180]: Test is failing for me, please check that you see it failing in Chromatic as well

File: src/components/Heading/__stories__/Heading.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+        tooltipProps={{
+          containerSelector: `#${OVERFLOW_TITLE_CONTAINER_ID}`
+        }}
+      >
+        Heading with ellipsis and tooltip when hovering
+      </Heading>
+      <div>
+        <Heading type={Heading.types.H2} ellipsis withoutTooltip maxLines={2}>
+          Heading with two lines overflow heading with 2 lines overflow heading with two lines overflow heading with 2
+          lines overflow heading with two lines overflow heading with two lines overflow heading with two lines overflow
```
Comment (SergeyRoyt) [inline src/components/Heading/__stories__/Heading.stories.mdx:205]: Maybe add here "without tooltip"?

File: src/components/LegacyHeading/__stories__/LegacyHeading.stories.mdx [hunk 5]
```diff
@@ -195,13 +197,13 @@ Our heading component support text highlight.
 
 <Canvas>
   <Story name="Empty state title">
-    <div className={styles["empty-state-container"]} aria-labelledby="empty-state-id">
```
Comment (SergeyRoyt) [inline src/components/LegacyHeading/__stories__/LegacyHeading.stories.mdx:198]: Is this class can be removed from the styles?

File: src/components/Text/__stories__/Text.stories.mdx [hunk 6]
```diff
@@ -41,25 +45,33 @@ export const textTemplate = createComponentTemplate(Text);
...
+  <Story name="Overview" args={{ children: "Hi, I'm a text!" }}>
     {textTemplate.bind({})}
   </Story>
 </Canvas>
 
+<Tip>
+  Check out our{" "}
+  <Link href="/?path=/docs/text-heading--overview" size={Link.sizes.SMALL}>
+    Heading
+  </Link>{" "}
```
Comment (SergeyRoyt) [inline src/components/Text/__stories__/Text.stories.mdx:57]: Double spacing

File: src/components/Text/__stories__/Text.stories.mdx
Comment (SergeyRoyt) [inline src/components/Text/__stories__/Text.stories.mdx:1]: - [Use cases and examples](#use-cases-and-examples) can be removed from the navigation - we don't have it here

File: webpack/published-ts-components.js [hunk 7]
```diff
@@ -3,7 +3,7 @@ const publishedTSComponents = {
   // plop_marker:published-components
   Badge: "components/Badge/Badge",
   Text: "components/Text/Text",
-  Title: "components/Title/Title",
+  Title: "components/Heading/Heading",
```
Comment (SergeyRoyt) [inline webpack/published-ts-components.js:6]: It's still under Title name here? Is that ok?

Comment (SergeyRoyt) [reply webpack/published-ts-components.js:6]: Perhaps it should be mentioned somewhere?

**Review by SergeyRoyt — APPROVED** [review]
Small fixes & tests fixes needed, the rest seems great

File: webpack/published-ts-components.js (hunk 7 above)
Comment (hadasfa) [reply webpack/published-ts-components.js:6]: fixed

**Pull request merged**